ci-operator/step-registry/platform-external/cluster/aws/pre/platform-external-cluster-aws-pre-chain.yaml

```
@@ -0,0 +1,15 @@
+chain:
+  as: platform-external-cluster-aws-pre
+  steps:
+  - chain: platform-external-cluster-pre
+  - ref: platform-external-cluster-aws-install
+  - ref: platform-external-cluster-wait-for-api-bootstrap
+  - ref: platform-external-ccm-conf-aws
+  - ref: platform-external-ccm-deploy
```

What about this chain including three steps?
```suggestion
  - chain: platform-external-ccm-conf-aws-deploy
```